packages/element-lib/src/sidetree-v2/protocol/resolve.spec.js

```diff
@@ -48,10 +47,7 @@ describe('resolve', () => {
     });
 
     it('should not work if specified kid does not exist in did document', async () => {
-      const invalidCreatePayload = {
-        ...createPayload,
-        header: { ...createPayload.header, kid: '#primaryy' },
-      };
+      const invalidCreatePayload = changeKid(createPayload, '#primaryy');
```

Comment:
nit: spelling?

Reply:
That's to create a payload with an invalid kid. I will rename to make that more explicit